pcsx2/CDVD/CDVD.cpp

```diff
@@ -2656,7 +2833,7 @@ static void cdvdWrite16(u8 rt) // SCOMMAND
 				cdvd.SCMDResult[12] = 0x9b;
 				break;
 
-			case 0x85: // secrman: __mechacon_auth_0x85
+			case 0x85: // secrman: __mechacon_auth_0x85, supported by PS3 CECH-A ps2emu
```

Comment:
delete the second part of this comment please.

Reply:
Could you, please, provide more information about why PS3 is restricted to mention?

Reply:
Because filling our code with "We read how to do this with sony's ps3 emu" is a really fucking dumb idea.